src/Datasource/EntityTrait.php

```diff
@@ -785,11 +785,11 @@ public function getDirty()
     {
         return array_keys($this->_dirty);
     }
-    
+
     /**
      * return value if property exists otherwise throw Exception
      *
-     * @param string $property
+     * @param string $property the field to get value for
```

Comment:
Sentences start with capital letter and end with fullstop.

    The field to get value for.

Reply:
I am sorry, I missed the most important part 👍  :-P